packages/main/src/components/AnalyticalTable/index.tsx

```diff
@@ -289,7 +289,7 @@ export interface TableProps extends Omit<CommonProps, 'title'> {
   LoadingComponent?: ComponentType<any>;
 }
 
-const useStyles = createComponentStyles(styles, { name: 'AnalyticalTable' });
+export const useStyles = createComponentStyles(styles, { name: 'AnalyticalTable' });
```

Comment:
I think you don't need this export, otherwise you will create all css classes twice.